Source/Laser/LaserEnvelope.H

```diff
@@ -0,0 +1,13 @@
+#include <complex>
+#include <iostream>
+#include <complex>
+#include <vector>
+
+
+#include <iostream>
+#include <vector>
+
+class LaserEnvelope {
+private:
+    amrex::Vector< amrex::FabArray< amrex::BaseFab <Complex> > > A_laser_envelope;
```

Comment:
All the variables that are read from the input file would need to be added here. For instance:
```
amrex::Real m_wavelength;
```